plugin/src/test/groovy/org/gradle/testretry/testframework/BaseTestNGFuncTest.groovy

```diff
@@ -205,7 +244,7 @@ class TestNGFuncTest extends AbstractFrameworkFuncTest {
 
             public class OrderedTests {
                 @Test(dependsOnMethods = {"childTest"})
-                public void grandchildTest() {}
+                public void grandChildTest() {}
```

Comment:
Afaik grandchild is a single word, so the case could be made to leave the casing as-is

Reply:
Agree, it is hard to make assertions, though, because `childTest` matches both the `childTest` and the `grandchildTest`.